### PR TITLE
Update create-crc-app to current frontend-starter-app

### PR DESCRIPTION
**Pull request by apinkert**
For [RHCLOUD-22676](https://issues.redhat.com/browse/RHCLOUD-22676). ~~`create-crc-app` will get a user almost all the way with its app creation. As of now, `npm run start` will throw a type error in `App.tsx` until the user registers their app.~~ This is no longer accurate, `create-crc-app` should work out of the box now! s/o to @Hyperkid123

### CODE REVIEW

File: packages/create-crc-app/templates/README.md [hunk 1]
```diff
@@ -36,8 +40,10 @@ Update `config/dev.webpack.config.js` according to your application URL. [Read m
 
 - The starter repo uses Travis to deploy the webpack build to another Github repo defined in `.travis.yml`
   - That Github repo has the following branches:
-    - `stage-beta` (deployed by pushing to `master` or `main` on this repo)
-    - `stage-stable` (deployed by pushing to `master/main-stable` on this repo)
+    - `ci-beta` (deployed by pushing to `master` or `main` on this repo)
+    - `ci-stable` (deployed by pushing to `ci-stable` on this repo)
+    - `qa-beta` (deployed by pushing to `qa-beta` on this repo)
+    - `qa-stable` (deployed by pushing to `qa-stable` on this repo)
```
Comment (aneelac22) [inline packages/create-crc-app/templates/README.md:46]: I don't think this change is right

File: packages/create-crc-app/templates/deploy/frontend.yaml [hunk 2]
```diff
@@ -29,4 +29,4 @@ parameters:
   - name: IMAGE_TAG
     required: true
   - name: IMAGE
-    value: TBD
+    value: TBD
```
Comment (aneelac22) [inline packages/create-crc-app/templates/deploy/frontend.yaml:32]: Can you add newline at the end of file

File: packages/create-crc-app/templates/.prettierrc.json [hunk 3]
```diff
@@ -1,9 +0,0 @@
-{
-  "printWidth": 150,
-  "arrowParens": "always",
-  "semi": true,
-  "tabWidth": 2,
-  "singleQuote": true,
-  "jsxSingleQuote": false,
-  "bracketSpacing": true
-}
```
Comment (aneelac22) [inline packages/create-crc-app/templates/.prettierrc.json:9]: This shouldn't be removed

File: packages/create-crc-app/templates/package.json [hunk 4]
```diff
@@ -1,48 +1,72 @@
...
+      "rimraf": "^3.0.2",
+      "stylelint": "13.13.1",
+      "stylelint-config-recommended-scss": "4.2.0",
+      "stylelint-scss": "3.19.0",
+      "typescript": "^4.7.4",
+      "webpack-bundle-analyzer": "4.4.2"
+  },
+  "insights": {
+    "appname": "{{appname}}"
+  }
```
Comment (aneelac22) [inline packages/create-crc-app/templates/package.json:71]: This will change when you add back prettier config

**Review by aneelac22 — CHANGES_REQUESTED** [review]
Looks good! Just some minor changes

File: packages/create-crc-app/templates/README.md (hunk 1 above)
Comment (apinkert) [reply packages/create-crc-app/templates/README.md]: I've removed this section as it doesn't apply to what's currently being generated by this package. I've also trimmed down the README to relevant (to this package) information.

File: packages/create-crc-app/templates/.prettierrc.json (hunk 3 above)
Comment (apinkert) [reply packages/create-crc-app/templates/.prettierrc.json:9]: Re-added

File: packages/create-crc-app/templates/src/App.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+import { Reducer } from 'redux';
+
+import Routes from './Routes';
+import './App.scss';
+
+import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities/Registry';
+import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/NotificationPortal';
+import { notificationsReducer } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import pckg from '../package.json';
```
Comment (Hyperkid123) [inline packages/create-crc-app/templates/src/App.tsx:12]: This import is not used.

File: packages/create-crc-app/templates/src/Components/SampleComponent/sample-component.test.tsx [hunk 6]
```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import '@testing-library/jest-dom';
```
Comment (Hyperkid123) [inline packages/create-crc-app/templates/src/Components/SampleComponent/sample-component.test.tsx:2]: This import is required to add additional assertions and other test methods.

File: packages/create-crc-app/templates/src/Routes/SamplePage/sample-page.scss [hunk 7]
```diff
@@ -0,0 +1,2 @@
+// js generates class '.page__{rootClass}' that wraps the content in your page added to the root tag
```
Comment (Hyperkid123) [inline packages/create-crc-app/templates/src/Routes/SamplePage/sample-page.scss:1]: Can you remove this file as a whole? Its no longer true.

File: packages/create-crc-app/templates/src/types/index.d.ts [hunk 8]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-var */
+import ChromeApi from '@redhat-cloud-services/frontend-components/ChromeApi';
```
Comment (Hyperkid123) [inline packages/create-crc-app/templates/src/types/index.d.ts:2]: This should not be necessary :thinking: the types package should be settings this up already.

**Review by aneelac22 — APPROVED** [review]

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**